src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -172,23 +172,6 @@ static jboolean
 compatible_versions(jint major_runtime,     jint minor_runtime,
                     jint major_compiletime, jint minor_compiletime)
 {
-    /*
-     * First check to see if versions are explicitly compatible via the
-     * list specified above.
-     */
-    int i;
-    for (i = 0; i < nof_compatible_versions; ++i) {
```

Comment:
nof_compatible_versions and compatible_versions_list variables are not used now and can be deleted

Reply:
Yes, I meant to delete all that. Fixed now.